submissions/hadizaSaadu/html-forms/contact-page.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<html>
+  <meta data="UTF-8" />
+  <head>
+    <title>contactform</title>
+    <body>
+      <form>
```

Comment:
The `<body>` tag should be outside the `<head>` tag

Reply:
> The `<body>` tag should be outside the `<head>` tag

ooooopppss...sorry about that, i have effected the corrections and re-pushed...thanks